frontend/talentsearch/src/js/components/Layout.tsx

```diff
@@ -137,61 +124,63 @@ const Layout = () => {
   }
 
   return (
-    <>
-      <Helmet>
-        <html lang={locale} />
-        <title>
+    <AppInsightsContextProvider>
```

Comment:
Hmm, I wonder if it would make sense to instead have this in the common context sandwich.  Then it would be along with the other context provders like the client provider and the locale provider.  It would make it available to the other two workspaces, too.

Reply:
I hope I've addressed this. Let me know if that isn't what you meant.